addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -1778,50 +1776,54 @@ const Wysiwyg = Widget.extend({
                     this.odooEditor.execCommand('setTag', 'pre');
                 },
             },
-            {
-                groupName: 'Navigation',
-                title: 'Link',
-                description: 'Add a link.',
-                fontawesome: 'fa-link',
-                callback: () => {
-                    this.toggleLinkTools({forceDialog: true});
-                },
-            },
-            {
-                groupName: 'Navigation',
-                title: 'Button',
-                description: 'Add a button.',
-                fontawesome: 'fa-link',
-                callback: () => {
-                    this.toggleLinkTools({forceDialog: true});
-                    // Force the button style after the link modal is open.
-                    setTimeout(() => {
-                        $(".o_link_dialog .link-style[value=primary]").click();
-                    }, 150);
-                },
-            },
         ];
-        if (options.isInternalUser) {
-            commands.push({
-                groupName: 'Medias',
-                title: 'Image',
-                description: 'Insert an image.',
-                fontawesome: 'fa-file-image-o',
-                callback: () => {
-                    this.openMediaDialog();
+        if (!this.options.plugins || this.options.plugins.LinkPlugin !== false) {
+            commands.push(
+                {
+                    groupName: 'Navigation',
+                    title: 'Link',
+                    description: 'Add a link.',
+                    fontawesome: 'fa-link',
+                    callback: () => {
+                        this.toggleLinkTools({forceDialog: true});
+                    },
                 },
-            });
+                {
+                    groupName: 'Navigation',
+                    title: 'Button',
+                    description: 'Add a button.',
+                    fontawesome: 'fa-link',
+                    callback: () => {
+                        this.toggleLinkTools({forceDialog: true});
+                        // Force the button style after the link modal is open.
+                        setTimeout(() => {
+                            $(".o_link_dialog .link-style[value=primary]").click();
+                        }, 150);
+                    },
+                },
+            );
         }
-        if (options.allowCommandVideo) {
+        if (!this.options.plugins || this.options.plugins.MediaPlugin !== false) {
```

Comment:
The `options.plugins.MediaPlugin` and `options.plugins.LinkPlugin` are not used anymore. Could we rename the `options.plugins.MediaPlugin` and `options.plugins.LinkPlugin` with `option.allowCommandImage` and `option.allowCommandLink`?

Reply:
This should be fixed now :+1: